main.py

```diff
@@ -1,10 +1,46 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with (open("players.json", "r")) as players_file:
+        players = json.load(players_file)
+
+    for nickname, player in players.items():
+        guild = player["guild"]
+        race = player["race"]
+        guild_id = None
+        race_id = None
+
+        if guild:
+            guild_db = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"]
+            )
+            guild_id = guild_db[0].id
```

Comment:
You can unpuck your rusult into a two viriables , one of them will be instance and second is `created` with can be `_ `, cuz we do not use it . That way it will be more clean imho